### PR TITLE
frontend: use js-cookie for managing cookies

### LINKED ISSUE #21: frontend: use js-cookie for storing & retrieving of cookies
js-cookie lib will help with storing and retrieving of information to and from cookies. 

https://github.com/js-cookie/js-cookie 

It will easy up the testing as it will not relay on the global state of window.

### PR DESCRIPTION
**Pull request by IvelinYR**
js-cookie library is now used for management of the cookies. 

Fixes #21

### CODE REVIEW

File: frontend/src/action/transaction.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ export function deposit(data) {
         type: 'DEPOSIT',
         payload: {
             request: {
-                method: 'post',
+                method: 'patch',
```
Comment (mgenov) [inline frontend/src/action/transaction.js]: It's better post to be used for deposit as it creates a new accounting operation. Keep in mind that Name of the PR doesn't have information information regarding these changes.

File: frontend/build/asset-manifest.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (mgenov) [inline frontend/build/asset-manifest.json]: build folder should be in .gitignore

File: frontend/src/reducers/login.js [hunk 3]
```diff
@@ -1,20 +1,12 @@
 import {browserHistory} from 'react-router';
+import * as Cookies from "js-cookie";
```
Comment (mgenov) [inline frontend/src/reducers/login.js:2]: import cookie from 'js-cookie' looks simpler.

**Review by mgenov — CHANGES_REQUESTED** [review]

**Comment by mgenov** [comment]
Note that PR title and body are updated, so you can update and commit message regarding the changes. 

You are able to close issue automatically by adding link to it in the body of the commit. 
https://help.github.com/articles/closing-issues-via-commit-messages/

This is why the commit message (and body of PR) has text: Fixes #21

**Comment by IvelinYR** [comment]
I'm ready with the changes

File: frontend/src/reducers/login.js (hunk 3 above)
Comment (mgenov) [reply frontend/src/reducers/login.js:2]: .

File: frontend/src/components/TransactionList/TransactionItems.js [hunk 4]
```diff
@@ -8,13 +8,13 @@ export default class TransactionItems extends React.Component {
...
 
-        if (this.props.transaction.type === "Withdrawal") {
+        if (this.props.transaction.TransactionType === "Withdrawal") {
             prefix = "-";
         }
         return (
             <tr>
-                <th >{transaction.type}</th>
-                <th >{transaction.date}</th>
+                <th >{transaction.TransactionType}</th>
```
Comment (mgenov) [inline frontend/src/components/TransactionList/TransactionItems.js]: transaction.TransactionType looks like a naming duplication. 

http://blog.thecodewhisperer.com/images/a-model-for-improving-names/improving-names.png

**Comment by mgenov** [comment]
Commit message is still the old one.

**Comment by IvelinYR** [comment]
All changes requested are implemented.

**Review by mgenov — APPROVED** [review]
LGTM !

**Pull request merged**